app/views/dashboards/_dashboard_article.html.erb

```diff
@@ -64,6 +64,7 @@
     <% if @current_user_pro %>
       <a href="<%= article.path %>/stats" data-no-instant class="cta pill blue">STATS</a>
     <% end %>
+    <a href="<%= article.path %>/subscriptions" data-no-instant class="cta pill">SUBSCRIPTIONS</a>
```

Comment:
Out of scope of this PR probably, but I think we should consider having a css class that is uppercased (if we don't already) and applying it here. We have lots of variations of different text formatting (lowercase, capitalized, all uppercase), and using classes would make this a bit easier/more robust.

Reply:
We do have one: `text-uppercase`. I think you can easily ignore this one - this view is still being considered as "legacy, pre-crayons era" type of view. Once I update its design, we will no longer have this type of buttons with uppercase labels. 

In this case I just wanted to keep it consistent with what's already there :)